app/renderer/reducers/frameReducer.js

```diff
@@ -92,7 +92,7 @@ const frameReducer = (state, action, immutableAction) => {
         }
       }
 
-      const title = immutableAction.getIn(['changeInfo', 'title'])
+      const title = immutableAction.getIn(['tabValue', 'title'])
```

Comment:
Currently the only problem with the title not updating is when you use navigation buttons, so when tab state changes. What should I wrote in todo? `TODO fix race condition with appState`